src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderViewMenu.vue

```diff
@@ -55,6 +55,14 @@ export default {
 				id: 'dayGridMonth',
 				icon: 'icon-view-module',
 				label: this.$t('calendar', 'Month'),
+			}, {
+				id: 'listWeek',
```

Comment:
There shouldn't be two list views. Please remove `listWeek` and keep `listMonth`.

Reply:
Why? Do you think the list per week is unnecessary? If so, should I remove mentions to it from everywhere in the code?

Reply:
> Why? Do you think the list per week is unnecessary?

Yes, it does not have any real advantage over list month. The information displayed and the presentation is identical.

Reply:
Seems fair, done.